atlp-rwanda/knights-ecomm-fe
# feat(landing-page): implementing landing page for client (buyer)

#### What does this PR do?
This PR implements the UI design for the landing page that introduces our brand and products.

#### Description of Task to be completed?
Tasks included:
- Design and implement a responsive landing page.
- Include a hero section and a CTA button.
- Add a featured products navigation menu (including categories).
- Ensure responsiveness on all devices.
- Implement state management with Redux.
- Display the banner for promotions on products.
- Ensure smooth API integration for data fetching.
- Show the Wishlist icon on products when the user is logged in.
- unit tests to ensure code quality

#### How should this be manually tested?
- Ensure You have backend service running and set VITE_APP_API_URL in your .env file.
- Clone this repository and navigate to the project directory
  ```bash
  git clone -b ft-register https://github.com/atlp-rwanda/knights-ecomm-fe.git
  cd knights-ecomm-fe
  git checkout ft-landing-page
  npm i
  npm run dev
  ```
#### What are the relevant pivotal tracker/Trello stories?
#41 

#### Screenshots
![image](https://github.com/atlp-rwanda/knights-ecomm-fe/assets/82057476/3f8f4918-59d1-4859-81b8-98ae878a877b)
